share/functions/funced.fish

```diff
@@ -102,14 +102,28 @@ function funced --description 'Edit function definition'
     else
         echo $init >$tmpname
     end
+
     # Repeatedly edit until it either parses successfully, or the user cancels
     # If the editor command itself fails, we assume the user cancelled or the file
     # could not be edited, and we do not try again
     while true
+        if which md5sum > /dev/null
```

Comment:
Please don't use `which` since that itself is not guaranteed to be installed. If something needs to be a command (and I'd expect it would be here), use `command -sq`. If it doesn't need to be, use `type -q`.